
Start -- Nov 10 16:33:01 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:33:01	Initing processor, tokenizer, and speaker_model
16:33:04	Loading Remote Dataset: facebook/voxpopuli. Sub-collection: en_accented
16:36:31	Finish Loading Remote Dataset. Length of dataset: 8387
16:36:31	Start Setting Sampling Rate to 16 kHz...
16:36:31	Setting Sampling Rate Successfully
16:36:31	Exception: <class 'TypeError'>. filter_and_prepare_dataset() missing 1 required positional argument: 'dataset'. Traceback: <traceback object at 0x7f1be4598dc0>

Start -- Nov 10 16:36:46 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:36:46	Initializing processor, tokenizer, and speaker_model
16:36:47	Loading Remote Dataset: facebook/voxpopuli. Sub-collection: en_accented
16:37:12	Finish Loading Remote Dataset. Length of dataset: 8387
16:37:12	Start Setting Sampling Rate to 16 kHz...
16:37:12	Setting Sampling Rate Successfully
16:37:12	Start Filtering Short Data
16:37:12	8358 data entries left after filtering
16:37:12	Start Preparing Dataset
16:38:41	KeyboardInterrupt detected. Exiting...

Start -- Nov 10 16:38:47 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:38:47	Initializing processor, tokenizer, and speaker_model
16:38:48	Loading Remote Dataset: facebook/voxpopuli. Sub-collection: en_accented
16:39:13	Finish Loading Remote Dataset. Length of dataset: 8387
16:39:13	Start Setting Sampling Rate to 16 kHz...
16:39:13	Setting Sampling Rate Successfully
16:39:13	Start Filtering Short Data
16:39:13	8358 data entries left after filtering
16:39:13	Start Preparing Dataset
16:40:04	Exception: <class 'multiprocess.context.TimeoutError'>. . Traceback: <traceback object at 0x7f1327d49f80>

Start -- Nov 10 16:40:08 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:40:08	Initializing processor, tokenizer, and speaker_model
16:40:09	Loading Remote Dataset: facebook/voxpopuli. Sub-collection: en_accented
16:40:34	Finish Loading Remote Dataset. Length of dataset: 8387
16:40:34	Start Setting Sampling Rate to 16 kHz...
16:40:34	Setting Sampling Rate Successfully
16:40:34	Start Filtering Short Data
16:40:34	8358 data entries left after filtering
16:40:34	Start Preparing Dataset
16:41:04	Exception: <class 'multiprocess.context.TimeoutError'>. . Traceback: <traceback object at 0x7fa037fc5100>

Start -- Nov 10 16:41:07 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:41:07	Initializing processor, tokenizer, and speaker_model
16:41:08	Loading Remote Dataset: facebook/voxpopuli. Sub-collection: en_accented
16:41:33	Finish Loading Remote Dataset. Length of dataset: 8387
16:41:33	Start Setting Sampling Rate to 16 kHz...
16:41:33	Setting Sampling Rate Successfully
16:41:33	Start Filtering Short Data
16:41:33	8358 data entries left after filtering
16:41:33	Start Preparing Dataset
16:47:00	Preparing dataset finished successfully
16:47:00	Start Filtering Long Data
16:47:00	6329 data entries left after filtering
16:47:00	save_processed_dataset is True. Saving processed dataset to dir: ./data/


Start -- Nov 10 16:50:43 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:50:43	Initializing processor, tokenizer, and speaker_model
16:50:44	Using Locally Processed Dataset. Skip Processing...
16:50:44	Failed to load local dataset. Please double check file exists and path is correct

Start -- Nov 10 16:51:12 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:51:12	Initializing processor, tokenizer, and speaker_model
16:51:12	Using Locally Processed Dataset. Skip Processing...
16:51:12	Failed to load local dataset. Please double check file exists and path is correct

Start -- Nov 10 16:52:45 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:52:45	Initializing processor, tokenizer, and speaker_model
16:52:45	Using Locally Processed Dataset. Skip Processing...
16:52:45	Failed to load local dataset. Please double check file exists and path is correct. You are trying to load a dataset that was saved using `save_to_disk`. Please use `load_from_disk` instead.

Start -- Nov 10 16:53:20 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
16:53:20	Initializing processor, tokenizer, and speaker_model
16:53:21	Using Locally Processed Dataset. Skip Processing...


Start -- Nov 10 17:42:43 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
17:42:43	Initializing processor, tokenizer, and speaker_model
17:43:04	Using Locally Processed Dataset. Skip Processing...
17:43:04	Generating Seq2SeqTrainer Arguments
17:43:04	Generating Seq2SeqTrainer
17:44:39	Exception: <class 'huggingface_hub.utils._errors.BadRequestError'>.  (Request ID: Root=1-654eb213-4bedca92381c9f61443bf392;dfd56855-90c4-42ec-bffa-f445e26543e3)

Bad request for commit endpoint:
"language[0]" with value "en_accented" is not valid. It must be an ISO 639-1, 639-2 or 639-3 code (two/three letters), or a special value like "code", "multilingual". If you want to use BCP-47 identifiers, you can specify them in language_bcp47.
"tags[0]" is not allowed to be empty. Traceback: <traceback object at 0x7eff3ca4d400>

Start -- Nov 10 17:46:04 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
17:46:04	Initializing processor, tokenizer, and speaker_model
17:46:06	Using Locally Processed Dataset. Skip Processing...
17:46:06	Generating Seq2SeqTrainer Arguments
17:46:06	Generating Seq2SeqTrainer
17:46:08	Start Training...
18:44:14	Start Saving the model...
18:44:17	Exception: <class 'KeyError'>. "Column test not in the dataset. Current columns in the dataset: ['input_ids', 'labels', 'speaker_embeddings']". Traceback: <traceback object at 0x7fc01cdca2c0>

Start -- Nov 10 20:53:15 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
20:53:15	Initializing processor, tokenizer, and speaker_model
20:53:17	Using Locally Processed Dataset. Skip Processing...
20:53:17	Loading Locally Stored Model from: ./model/trained_yifanhua-threadripper20:53:15
20:53:17	Exception: <class 'huggingface_hub.utils._validators.HFValidationError'>. Repo id must be in the form 'repo_name' or 'namespace/repo_name': './model/trained_yifanhua-threadripper20:53:15'. Use `repo_type` argument if needed.. Traceback: <traceback object at 0x7fca353cd380>

Start -- Nov 10 20:54:20 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
20:54:20	Initializing processor, tokenizer, and speaker_model
20:54:23	Using Locally Processed Dataset. Skip Processing...
20:54:23	Loading Locally Stored Model from: ./model/trained_yifanhua-threadripper20:54:20
20:54:23	Exception: <class 'huggingface_hub.utils._validators.HFValidationError'>. Repo id must be in the form 'repo_name' or 'namespace/repo_name': './model/trained_yifanhua-threadripper20:54:20'. Use `repo_type` argument if needed.. Traceback: <traceback object at 0x7fd3a2a8e1c0>

Start -- Nov 10 21:05:37 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:05:37	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:05:39	Using Locally Processed Dataset. Skip Processing...
21:05:39	Loading Locally Stored Model from: ./model/trained_yifanhua-threadripper21:03:24
21:05:51	KeyboardInterrupt detected. Exiting...

Start -- Nov 10 21:06:31 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:06:31	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:06:33	Using Locally Processed Dataset. Skip Processing...
21:06:33	Loading Locally Stored Model from: ./model/trained_yifanhua-threadripper21:06:31
21:06:36	Exception: <class 'IndexError'>. list index out of range. Traceback: <traceback object at 0x7fb5cb4a8380>

Start -- Nov 10 21:07:53 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:07:53	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:07:55	Using Locally Processed Dataset. Skip Processing...
21:07:55	Loading Locally Stored Model from: ./model/
21:08:01	Exception: <class 'huggingface_hub.utils._validators.HFValidationError'>. Repo id must be in the form 'repo_name' or 'namespace/repo_name': './model/trained_yifanhua-threadripper21:07:53'. Use `repo_type` argument if needed.. Traceback: <traceback object at 0x7fba6c0c8180>

Start -- Nov 10 21:08:30 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:08:30	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:08:32	Using Locally Processed Dataset. Skip Processing...
21:08:32	Loading Locally Stored Model from: ./model/
21:09:18	Exception: <class 'huggingface_hub.utils._validators.HFValidationError'>. Repo id must be in the form 'repo_name' or 'namespace/repo_name': './model/trained_yifanhua-threadripper21:08:30'. Use `repo_type` argument if needed.. Traceback: <traceback object at 0x7f064e128240>

Start -- Nov 10 21:10:48 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:10:48	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:10:50	Using Locally Processed Dataset. Skip Processing...
21:10:50	Loading Locally Stored Model from: ./model/
21:10:51	Using model: ./model/trained_yifanhua-threadripper17:46:04
21:10:51	Exception: <class 'huggingface_hub.utils._validators.HFValidationError'>. Repo id must be in the form 'repo_name' or 'namespace/repo_name': './model/trained_yifanhua-threadripper21:10:48'. Use `repo_type` argument if needed.. Traceback: <traceback object at 0x7f3dcffd6b40>

Start -- Nov 10 21:11:12 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:11:12	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:11:14	Using Locally Processed Dataset. Skip Processing...
21:11:14	Loading Locally Stored Model from: ./model/
21:11:16	Using model: ./model/trained_yifanhua-threadripper17:46:04


Start -- Nov 10 21:23:19 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:23:19	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:23:22	Using Locally Processed Dataset. Skip Processing...
21:23:22	Exception: <class 'NameError'>. name 'selected_model' is not defined. Traceback: <traceback object at 0x7fb9d05c8c00>

Start -- Nov 10 21:23:42 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:23:42	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:23:44	Using Locally Processed Dataset. Skip Processing...
21:23:44	Using model: ./model/trained_yifanhua-threadripper17:46:04


Start -- Nov 10 21:24:20 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:24:20	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:24:23	Using Locally Processed Dataset. Skip Processing...
21:24:23	Using model: ./model/trained_yifanhua-threadripper17:46:04


Start -- Nov 10 21:24:40 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:24:40	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:24:43	Using Locally Processed Dataset. Skip Processing...
21:24:43	Using model: ./model/trained_yifanhua-threadripper17:46:04


Start -- Nov 10 21:25:05 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:25:05	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:25:07	Using Locally Processed Dataset. Skip Processing...
21:25:07	Using model: ./model/trained_yifanhua-threadripper17:46:04


Start -- Nov 10 21:31:59 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:31:59	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:32:02	Using Locally Processed Dataset. Skip Processing...
21:32:02	Using model: ./model/trained_yifanhua-threadripper17:46:04


Start -- Nov 10 21:33:06 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:33:06	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:33:10	Using Locally Processed Dataset. Skip Processing...
21:33:10	Generating Seq2SeqTrainer Arguments
21:33:10	Generating Seq2SeqTrainer
21:33:10	Start Training...
21:33:11	Exception: <class 'RuntimeError'>. cuDNN error: CUDNN_STATUS_NOT_INITIALIZED. Traceback: <traceback object at 0x7f4a3c31cdc0>

Start -- Nov 10 21:34:38 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:34:38	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:34:40	Using Locally Processed Dataset. Skip Processing...
21:34:40	Generating Seq2SeqTrainer Arguments
21:34:40	Generating Seq2SeqTrainer
21:34:40	Start Training...
21:34:43	Exception: <class 'torch.cuda.OutOfMemoryError'>. CUDA out of memory. Tried to allocate 268.00 MiB. GPU 0 has a total capacty of 7.75 GiB of which 109.75 MiB is free. Including non-PyTorch memory, this process has 5.15 GiB memory in use. Of the allocated memory 4.51 GiB is allocated by PyTorch, and 503.43 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF. Traceback: <traceback object at 0x7f1c6e0c6140>

Start -- Nov 10 21:38:17 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:38:17	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:38:19	Using Locally Processed Dataset. Skip Processing...
21:38:19	Generating Seq2SeqTrainer Arguments
21:38:19	Generating Seq2SeqTrainer
21:38:19	Start Training...
21:38:20	Exception: <class 'torch.cuda.OutOfMemoryError'>. CUDA out of memory. Tried to allocate 274.00 MiB. GPU 0 has a total capacty of 7.75 GiB of which 311.38 MiB is free. Including non-PyTorch memory, this process has 4.58 GiB memory in use. Of the allocated memory 4.08 GiB is allocated by PyTorch, and 391.68 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF. Traceback: <traceback object at 0x7f81ea7d94c0>

Start -- Nov 10 21:39:25 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
21:39:25	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
21:39:27	Using Locally Processed Dataset. Skip Processing...
21:39:27	Generating Seq2SeqTrainer Arguments
21:39:27	Generating Seq2SeqTrainer
21:39:27	Start Training...
22:16:58	Start Saving the model...
22:17:05	Exception: <class 'RuntimeError'>. Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu! (when checking argument for argument index in method wrapper_CUDA__index_select). Traceback: <traceback object at 0x7f606a7c6ec0>

Start -- Nov 10 22:18:28 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:18:28	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:18:31	Using Locally Processed Dataset. Skip Processing...
22:18:34	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:18:35	Exception: <class 'RuntimeError'>. Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument tensors in method wrapper_CUDA_cat). Traceback: <traceback object at 0x7fa94bf6ecc0>

Start -- Nov 10 22:19:14 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:19:14	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:19:16	Using Locally Processed Dataset. Skip Processing...
22:19:50	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:19:51	Exception: <class 'RuntimeError'>. Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper_CUDA__index_select). Traceback: <traceback object at 0x7fa9c04cc4c0>

Start -- Nov 10 22:27:51 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:27:51	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:27:53	Using Locally Processed Dataset. Skip Processing...
22:27:56	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 22:28:09 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:28:09	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:28:11	Using Locally Processed Dataset. Skip Processing...
22:28:14	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 22:28:34 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:28:34	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:28:37	Using Locally Processed Dataset. Skip Processing...
22:28:39	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 22:31:52 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:31:52	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:31:54	Using Locally Processed Dataset. Skip Processing...
22:31:54	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 22:33:35 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:33:35	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:33:37	Using Locally Processed Dataset. Skip Processing...
22:33:45	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 22:34:16 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:34:16	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:34:18	Using Locally Processed Dataset. Skip Processing...
22:37:12	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 22:37:41 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:37:41	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:37:43	Using Locally Processed Dataset. Skip Processing...
22:39:21	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 22:39:39 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:39:39	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:39:41	Using Locally Processed Dataset. Skip Processing...
22:39:52	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 22:40:05 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:40:05	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:40:07	Using Locally Processed Dataset. Skip Processing...
22:40:08	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:40:09	Exception: ['Traceback (most recent call last):\n', '  File "/home/yifanhua/5455-term-project/src/main.py", line 374, in <module>\n    spectrogram = pretrained_model.generate_speech(inputs["input_ids"], speaker_embeddings)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context\n    return func(*args, **kwargs)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 2824, in generate_speech\n    return _generate_speech(\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 2480, in _generate_speech\n    encoder_out = model.speecht5.encoder(\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl\n    return forward_call(*args, **kwargs)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 1448, in forward\n    hidden_states = self.prenet(input_values)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl\n    return forward_call(*args, **kwargs)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 787, in forward\n    inputs_embeds = self.embed_tokens(input_ids)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl\n    return self._call_impl(*args, **kwargs)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl\n    return forward_call(*args, **kwargs)\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/sparse.py", line 162, in forward\n    return F.embedding(\n', '  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/functional.py", line 2233, in embedding\n    return torch.embedding(weight, input, padding_idx, scale_grad_by_freq, sparse)\n', 'RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper_CUDA__index_select)\n']

Start -- Nov 10 22:43:07 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:43:07	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:43:09	Using Locally Processed Dataset. Skip Processing...
22:43:14	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:43:15	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 374, in <module>
    spectrogram = pretrained_model.generate_speech(inputs["input_ids"], speaker_embeddings)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 2824, in generate_speech
    return _generate_speech(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 2480, in _generate_speech
    encoder_out = model.speecht5.encoder(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 1448, in forward
    hidden_states = self.prenet(input_values)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 787, in forward
    inputs_embeds = self.embed_tokens(input_ids)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/sparse.py", line 162, in forward
    return F.embedding(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/functional.py", line 2233, in embedding
    return torch.embedding(weight, input, padding_idx, scale_grad_by_freq, sparse)
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper_CUDA__index_select)


Start -- Nov 10 22:46:38 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:46:38	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:46:40	Using Locally Processed Dataset. Skip Processing...
22:46:54	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:46:55	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 374, in <module>
    spectrogram = pretrained_model.generate_speech(inputs["input_ids"], speaker_embeddings)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 2824, in generate_speech
    return _generate_speech(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 2480, in _generate_speech
    encoder_out = model.speecht5.encoder(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 1448, in forward
    hidden_states = self.prenet(input_values)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 787, in forward
    inputs_embeds = self.embed_tokens(input_ids)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/sparse.py", line 162, in forward
    return F.embedding(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/functional.py", line 2233, in embedding
    return torch.embedding(weight, input, padding_idx, scale_grad_by_freq, sparse)
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper_CUDA__index_select)


Start -- Nov 10 22:47:34 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:47:34	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:47:36	Using Locally Processed Dataset. Skip Processing...
22:47:37	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:47:38	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 373, in <module>
    spectrogram = pretrained_model.generate_speech(inputs["input_ids"], speaker_embeddings)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 2824, in generate_speech
    return _generate_speech(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 2480, in _generate_speech
    encoder_out = model.speecht5.encoder(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 1448, in forward
    hidden_states = self.prenet(input_values)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 787, in forward
    inputs_embeds = self.embed_tokens(input_ids)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/sparse.py", line 162, in forward
    return F.embedding(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/functional.py", line 2233, in embedding
    return torch.embedding(weight, input, padding_idx, scale_grad_by_freq, sparse)
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper_CUDA__index_select)


Start -- Nov 10 22:48:28 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:48:28	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:48:30	Using Locally Processed Dataset. Skip Processing...
22:48:41	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:48:42	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 376, in <module>
    speech = vocoder(spectrogram)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 3208, in forward
    spectrogram = (spectrogram - self.mean) / self.scale
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!


Start -- Nov 10 22:52:35 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:52:35	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:52:37	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 225, in <module>
    pretrained_model, processor, tokenizer, speaker_model, data_collator, vocoder = init_global_variables()
  File "/home/yifanhua/5455-term-project/src/main.py", line 86, in init_global_variables
    _processor = SpeechT5Processor.from_pretrained("microsoft/speecht5_tts").to(device)
AttributeError: 'SpeechT5Processor' object has no attribute 'to'


Start -- Nov 10 22:52:58 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:52:58	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:53:00	Using Locally Processed Dataset. Skip Processing...
22:53:02	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:53:04	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 376, in <module>
    speech = vocoder(spectrogram)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 3208, in forward
    spectrogram = (spectrogram - self.mean) / self.scale
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!


Start -- Nov 10 22:53:37 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:53:37	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:53:39	Using Locally Processed Dataset. Skip Processing...
22:53:40	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:53:41	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 376, in <module>
    speech = vocoder(spectrogram)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/models/speecht5/modeling_speecht5.py", line 3208, in forward
    spectrogram = (spectrogram - self.mean) / self.scale
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu!


Start -- Nov 10 22:53:58 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:53:58	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:54:00	Using Locally Processed Dataset. Skip Processing...
22:54:00	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:54:02	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 380, in <module>
    Audio(speech.numpy(), rate=16000)
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.


Start -- Nov 10 22:55:16 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:55:16	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:55:18	Using Locally Processed Dataset. Skip Processing...
22:55:19	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
22:55:21	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 384, in <module>
    sf.write("output.wav", speech.numpy(), samplerate=16000)
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.


Start -- Nov 10 22:58:12 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
22:58:12	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
22:58:15	Using Locally Processed Dataset. Skip Processing...
22:58:44	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 23:04:34 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
23:04:34	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
23:04:36	Using Locally Processed Dataset. Skip Processing...
23:04:38	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
23:04:39	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 371, in <module>
    example = divided_dataset["test"][5]
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 2803, in __getitem__
    return self._getitem(key)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 2787, in _getitem
    pa_subtable = query_table(self._data, key, indices=self._indices if self._indices is not None else None)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 583, in query_table
    _check_valid_index_key(key, size)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 526, in _check_valid_index_key
    raise IndexError(f"Invalid key: {key} is out of bounds for size {size}")
IndexError: Invalid key: 5 is out of bounds for size 5


Start -- Nov 10 23:04:49 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
23:04:49	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
23:04:51	Using Locally Processed Dataset. Skip Processing...
23:04:53	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39

Start -- Nov 10 23:06:29 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
23:06:29	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
23:06:32	Using Locally Processed Dataset. Skip Processing...
23:06:34	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
23:06:35	Input text: I'm loading the model from the Hugging Face Hub!
23:06:35	Using speaker embedding: tensor([[-7.6836e-02,  6.3403e-02,  4.0562e-02,  3.8820e-02, -1.7536e-02,
          9.0452e-03, -4.9579e-02,  4.5229e-02,  4.2024e-02,  1.4980e-02,
         -7.4401e-02, -8.2151e-02,  4.0258e-02,  3.4921e-02,  3.0220e-02,
          4.3113e-02,  2.8215e-02,  1.8833e-02,  1.4011e-02,  1.5947e-02,
          3.4128e-02,  2.5922e-02, -1.3152e-02, -5.4478e-02, -6.3750e-02,
         -9.2914e-03, -6.7767e-02,  1.5682e-03,  4.6559e-02,  4.0730e-02,
         -3.0536e-03,  5.2462e-02,  4.2241e-02,  1.7429e-03,  2.8580e-02,
         -7.5006e-02,  3.1938e-02,  2.8833e-02,  1.4967e-02, -6.4432e-02,
          1.8939e-02, -9.6601e-03,  2.1530e-02,  5.3108e-02,  4.4103e-02,
         -1.0370e-01, -1.8632e-02,  1.2255e-02, -9.0650e-02,  4.7486e-02,
          2.5308e-02,  2.9944e-02,  1.1476e-02,  3.2961e-02, -8.1554e-02,
         -6.5184e-02,  1.4976e-02,  2.7341e-02,  3.2480e-02,  2.3878e-02,
          1.6458e-02, -7.3298e-03, -1.4582e-02,  1.5723e-02,  3.7580e-02,
          2.4833e-02,  4.5336e-02, -5.6571e-02, -6.9297e-02, -4.3971e-02,
          5.3361e-04,  9.0129e-03,  4.7100e-02,  1.2022e-02,  2.4132e-02,
          4.5285e-02,  8.8045e-03,  2.0786e-02, -5.7390e-02, -6.9695e-02,
         -6.2979e-02, -5.5661e-02, -7.1943e-02, -5.0074e-02, -3.2374e-02,
         -7.4352e-02, -5.9192e-02,  4.9010e-02,  2.1572e-02, -6.5620e-02,
          2.7758e-02, -9.1477e-02,  1.7411e-02, -7.2112e-02,  3.7737e-02,
         -6.9570e-03,  3.6025e-02,  3.9992e-02, -5.8816e-02, -7.7995e-02,
          1.7029e-02, -7.5107e-02, -7.2432e-02,  2.5058e-02,  3.2963e-02,
          2.0084e-02,  4.3422e-02,  6.3396e-02,  2.6535e-02,  2.6281e-02,
         -9.0686e-02,  1.0709e-02,  7.4889e-02,  5.9660e-03,  2.3868e-02,
          5.3681e-02, -6.9733e-02,  5.0336e-03, -5.4502e-02, -1.8466e-03,
          1.7699e-02, -5.9915e-02,  6.1621e-03,  4.6774e-02, -6.2205e-02,
          2.2998e-02, -7.8370e-02,  2.9577e-02,  2.5580e-02,  5.9209e-02,
          2.0903e-02,  3.4895e-02,  1.0720e-02,  6.7392e-02,  2.8271e-02,
         -8.2322e-02, -9.4090e-02,  2.9570e-02, -6.0319e-02,  8.3229e-04,
          3.6323e-02,  1.4722e-03,  1.1924e-02,  6.2906e-02, -6.4020e-02,
          3.6470e-02, -1.3211e-02,  1.3893e-02,  3.0392e-02, -8.4153e-02,
          4.6257e-02, -6.4347e-02, -8.6102e-02,  3.2333e-02,  1.8153e-02,
          1.8537e-02,  2.5648e-02, -8.1804e-02, -7.1273e-02,  5.8943e-02,
          9.4322e-03,  2.1041e-02,  2.3579e-02,  3.5110e-02,  2.9283e-03,
          3.5936e-03,  4.1134e-02,  2.3648e-02,  2.4587e-02,  3.7277e-02,
          1.4844e-02, -6.3547e-02,  6.7236e-03, -7.0230e-02, -3.8990e-02,
          2.2931e-02, -6.3326e-02,  1.6781e-02,  5.9014e-02, -7.7587e-02,
          2.7868e-02, -7.6592e-02,  3.3783e-02,  1.9375e-02, -7.5908e-02,
         -1.0387e-02,  1.4482e-02,  3.2370e-02, -6.6226e-02, -6.2260e-02,
          2.3353e-02,  2.7653e-02, -7.1155e-02,  2.6955e-02,  1.1666e-02,
          2.7088e-02,  1.8777e-02,  1.5794e-02,  3.1896e-02,  5.7146e-03,
          1.7183e-02,  3.0124e-02, -4.7382e-02,  1.3569e-02,  2.6958e-02,
          1.3499e-02,  3.0649e-02,  2.4900e-02,  3.9258e-02,  2.0793e-02,
          3.9824e-03, -7.0943e-02, -5.0493e-02,  4.8370e-02, -6.2684e-02,
          3.4115e-02,  3.9837e-02, -9.1817e-02,  2.9612e-02,  3.2944e-02,
         -6.8549e-02,  3.3213e-02, -7.0979e-02, -3.4610e-02, -7.1876e-02,
          5.6086e-02,  2.0452e-02,  7.0578e-02,  6.3566e-02,  2.9136e-02,
         -1.3794e-02,  6.8589e-02,  2.3369e-02,  1.6955e-02,  6.2187e-03,
          7.6584e-03,  4.0822e-02,  1.8875e-02,  3.2420e-03,  4.5718e-02,
         -5.8132e-02,  2.0638e-02, -9.0662e-02,  2.6839e-02, -6.8926e-02,
          2.6182e-02,  3.1046e-02,  2.3971e-02,  1.9273e-02,  3.9320e-02,
          5.2058e-02,  2.1656e-02,  8.4990e-03, -7.2728e-02,  5.7367e-03,
          4.7709e-02,  1.8558e-02,  4.2686e-02, -4.6847e-02,  5.7891e-02,
          1.9858e-02, -7.3581e-02,  4.9841e-02,  2.3763e-02, -9.7366e-05,
          3.2634e-02,  4.0977e-02,  4.1435e-02,  2.3259e-02,  2.1570e-02,
          2.7965e-02, -3.6707e-03,  6.4330e-03,  4.3431e-02, -8.1272e-02,
          2.1740e-02,  3.4477e-02,  3.4442e-02, -5.8742e-02, -8.1472e-02,
         -2.7763e-02,  3.0170e-02,  5.8892e-02, -7.6027e-02,  3.4409e-02,
          4.3539e-02, -1.2113e-02,  4.0763e-02, -6.8863e-02,  5.1888e-03,
          7.8995e-03,  6.7038e-03,  2.1326e-02, -1.5543e-02,  3.6723e-02,
          2.3226e-02,  2.9182e-02, -9.0791e-03,  3.0437e-02,  5.0427e-02,
         -6.9418e-02,  2.7464e-02,  2.2207e-02, -7.3056e-02,  1.6306e-02,
          3.0963e-02,  2.9017e-02,  2.0750e-02,  3.1957e-02,  2.9078e-02,
          4.5587e-02,  2.0261e-02,  5.0033e-03,  5.8378e-02, -8.4611e-02,
         -7.6075e-02, -1.9838e-02,  4.0487e-02, -6.1888e-02,  4.3652e-02,
         -6.2916e-02,  3.9619e-02, -5.2359e-02,  6.3863e-03,  4.9324e-02,
         -6.9716e-02,  4.2202e-02,  6.4274e-02, -6.0595e-02, -5.6750e-02,
         -6.0675e-02,  2.8630e-02,  2.9104e-02,  4.5632e-02, -8.1962e-02,
          4.5834e-02,  2.8655e-02,  1.9608e-02,  3.6816e-03,  4.6190e-02,
          2.9040e-02, -4.9038e-02,  3.5908e-02,  2.0035e-03, -9.5161e-02,
          1.3396e-02,  3.1781e-02,  2.4335e-02, -5.2164e-02, -7.2463e-02,
         -6.3275e-02,  2.3019e-02, -7.1583e-02,  1.1089e-01,  2.1324e-02,
         -4.8902e-02, -5.4751e-03,  5.0892e-02, -3.2968e-02,  1.6456e-03,
         -1.0364e-01,  1.1794e-02,  2.8560e-03,  3.1179e-02,  5.8883e-02,
         -8.6580e-03, -5.3970e-02,  1.7495e-02,  2.7266e-02,  2.2890e-02,
          5.9171e-03,  3.1879e-02, -5.4221e-02,  7.6534e-03, -1.3825e-02,
         -2.2423e-03,  3.0243e-02,  2.1296e-02,  3.1641e-02,  1.9973e-02,
         -8.4289e-02, -1.0444e-01,  2.8702e-02,  3.7857e-02,  2.3716e-02,
          2.9539e-02,  5.5774e-02, -6.6566e-02,  2.0514e-02,  3.5840e-02,
          1.6014e-02,  1.7473e-02, -4.7530e-02,  2.7591e-02,  2.4982e-02,
         -8.2298e-03,  5.0999e-02, -7.7798e-02,  3.4287e-02, -1.2068e-02,
          1.9090e-03, -5.1682e-02,  9.6981e-03,  2.2473e-02,  3.1747e-02,
          4.2606e-03,  2.0931e-03, -6.1320e-02, -5.2187e-02, -7.9547e-02,
         -4.8807e-02,  3.5517e-02,  3.4056e-02, -1.2789e-02,  4.4212e-02,
         -6.5526e-02,  1.5182e-02, -7.6318e-02, -1.3027e-01, -1.9935e-02,
         -7.3741e-02, -1.0807e-03,  2.6086e-02,  2.6132e-02, -5.8830e-02,
          3.4185e-02,  4.0558e-02,  3.2187e-02,  4.5877e-02, -5.3099e-02,
         -4.2563e-02, -1.2752e-02, -1.9676e-02,  3.6419e-02,  3.3966e-02,
          2.8699e-03,  3.8468e-02,  1.4768e-02,  2.2580e-02, -7.2594e-02,
         -4.6226e-03,  3.8273e-02,  1.4515e-02,  2.8593e-03,  7.1926e-03,
          4.0886e-02,  1.9179e-02,  5.4359e-03,  4.3910e-03, -1.1557e-02,
          3.2949e-02,  1.9718e-02,  3.7592e-02,  4.9951e-02,  4.5965e-02,
          3.4425e-03,  3.8280e-02, -6.6107e-02, -5.9034e-02,  4.0041e-02,
         -1.1044e-02,  3.2420e-03,  2.6370e-02,  3.4795e-02, -7.8728e-02,
          2.9791e-02,  2.5478e-02,  9.7442e-03,  5.1081e-02,  3.3058e-02,
         -2.0060e-03,  4.1954e-02,  2.4819e-03,  5.9872e-03, -5.1906e-02,
         -8.1537e-02,  1.6202e-02, -4.1698e-03,  2.7178e-02,  4.4018e-02,
         -6.4413e-02, -5.9531e-02,  8.8259e-04,  3.8542e-02,  2.8775e-02,
          3.4858e-02, -6.3876e-02,  2.6297e-02, -3.1551e-02, -1.0353e-02,
         -3.4075e-02,  2.4226e-02, -3.8576e-02, -2.3174e-02, -3.1889e-02,
          1.6949e-02, -4.8726e-02,  4.1621e-03, -6.3748e-02, -7.0101e-02,
          7.8505e-03,  4.0942e-02, -6.4920e-02,  3.1049e-02,  5.1092e-03,
         -1.3265e-02,  1.5715e-02,  4.5942e-02,  6.0792e-02, -5.3153e-02,
          3.5526e-02, -6.5377e-02]], device='cuda:0')

Start -- Nov 10 23:08:00 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
23:08:00	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
23:08:03	Using Locally Processed Dataset. Skip Processing...
23:08:03	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
23:08:04	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 11 09:49:03 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
09:49:03	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
09:49:05	Loading Remote Dataset: facebook/voxpopuli. Sub-collection: en_accented
09:49:32	Finish Loading Remote Dataset. Length of dataset: 8387
09:49:32	Start Setting Sampling Rate to 16 kHz...
09:49:32	Setting Sampling Rate Successfully
09:49:32	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 364, in <module>
    sort_speaker()
TypeError: sort_speaker() missing 1 required positional argument: '_dataset'


Start -- Nov 11 09:49:49 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
09:49:49	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
09:49:51	Loading Remote Dataset: facebook/voxpopuli. Sub-collection: en_accented
09:50:17	Finish Loading Remote Dataset. Length of dataset: 8387
09:50:17	Start Setting Sampling Rate to 16 kHz...
09:50:17	Setting Sampling Rate Successfully
09:50:18	Start Filtering Short Data
09:50:18	8358 data entries left after filtering
09:50:18	Start Preparing Dataset
09:50:39	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 09:53:21 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
09:53:21	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
09:53:23	Loading Remote Dataset: facebook/voxpopuli. Sub-collection: en_accented
09:53:49	Finish Loading Remote Dataset. Length of dataset: 8387
09:53:49	Start Setting Sampling Rate to 16 kHz...
09:53:49	Setting Sampling Rate Successfully
09:53:49	Start Filtering Short Data
09:53:49	8358 data entries left after filtering
09:53:49	Start Preparing Dataset
09:53:55	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 11:47:35 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
11:47:35	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
11:47:38	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
12:22:59	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 397, in <module>
    # Step 1A: Process & Preparing Dataset
  File "/home/yifanhua/5455-term-project/src/main.py", line 265, in load_remote_dataset
    _dataset = load_dataset(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/load.py", line 2166, in load_dataset
    ds = builder_instance.as_dataset(split=split, verification_mode=verification_mode, in_memory=keep_in_memory)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/builder.py", line 1190, in as_dataset
    datasets = map_nested(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/utils/py_utils.py", line 456, in map_nested
    return function(data_struct)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/builder.py", line 1220, in _build_single_dataset
    ds = self._as_dataset(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/builder.py", line 1294, in _as_dataset
    dataset_kwargs = ArrowReader(cache_dir, self.info).read(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_reader.py", line 240, in read
    files = self.get_file_instructions(name, instructions, split_infos)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_reader.py", line 213, in get_file_instructions
    file_instructions = make_file_instructions(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_reader.py", line 130, in make_file_instructions
    absolute_instructions = instruction.to_absolute(name2len)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_reader.py", line 653, in to_absolute
    return [_rel_to_abs_instr(rel_instr, name2len) for rel_instr in self._relative_instructions]
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_reader.py", line 653, in <listcomp>
    return [_rel_to_abs_instr(rel_instr, name2len) for rel_instr in self._relative_instructions]
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_reader.py", line 465, in _rel_to_abs_instr
    raise ValueError(f'Unknown split "{split}". Should be one of {list(name2len)}.')
ValueError: Unknown split "validated". Should be one of ['train', 'test', 'validation', 'other', 'invalidated'].


Start -- Nov 11 12:23:47 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
12:23:47	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
12:23:50	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
12:27:15	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 12:27:19 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
12:27:19	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
12:27:22	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
12:32:09	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 13:32:36 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
13:32:36	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
13:32:38	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
13:33:01	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 13:45:53 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
13:45:53	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
13:45:56	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
13:46:07	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 13:46:47 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
13:46:47	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
13:46:50	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
13:58:24	Finish Loading Remote Dataset. Length of dataset: 12135
13:58:24	Begin Sharding the dataset
13:58:24	Total 252 shards
13:58:24	Using Mozilla Common Voice. Additional Cleaning Needed
13:58:24	Starting Size of Mozilla Common Voice: 12135
13:59:05	Size after filtering accent: 8671
13:59:08	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 407, in <module>
    dataset = load_remote_dataset()
  File "/home/yifanhua/5455-term-project/src/main.py", line 281, in load_remote_dataset
    _dataset = list(executor.map(clean_mozilla_dataset(_dataset), _shard_dataset))
  File "/home/yifanhua/5455-term-project/src/main.py", line 250, in clean_mozilla_dataset
    _dataset = _dataset.filter(lambda entry: entry["gender"].isin(['female', 'male']))
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 557, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/fingerprint.py", line 511, in wrapper
    out = func(dataset, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3627, in filter
    indices = self.map(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 592, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 557, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3097, in map
    for rank, done, content in Dataset._map_single(**dataset_kwargs):
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3474, in _map_single
    batch = apply_function_on_filtered_inputs(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3353, in apply_function_on_filtered_inputs
    processed_inputs = function(*fn_args, *additional_args, **fn_kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 6238, in get_indices_from_mask_function
    function(example, indices[i], **fn_kwargs) if with_indices else function(example, **fn_kwargs)
  File "/home/yifanhua/5455-term-project/src/main.py", line 250, in <lambda>
    _dataset = _dataset.filter(lambda entry: entry["gender"].isin(['female', 'male']))
AttributeError: 'str' object has no attribute 'isin'


Start -- Nov 11 13:59:29 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
13:59:29	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
13:59:32	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
13:59:32	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 14:02:27 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
14:02:27	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
14:02:29	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
14:14:06	Finish Loading Remote Dataset. Length of dataset: 12135
14:14:06	Using Mozilla Common Voice. Additional Cleaning Needed
14:14:06	Starting Size of Mozilla Common Voice: 12135
14:14:45	Size after filtering accent: 8671
14:15:12	Size after filtering gender: 8637
14:15:38	Size after filtering voting: 8637
14:15:38	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 410, in <module>
    dataset = load_remote_dataset()
  File "/home/yifanhua/5455-term-project/src/main.py", line 285, in load_remote_dataset
    _dataset = clean_mozilla_dataset(_dataset)
  File "/home/yifanhua/5455-term-project/src/main.py", line 256, in clean_mozilla_dataset
    _dataset = _dataset.map(lambda entry: entry['sentence'].str.lower())
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 592, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 557, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3097, in map
    for rank, done, content in Dataset._map_single(**dataset_kwargs):
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3450, in _map_single
    example = apply_function_on_filtered_inputs(example, i, offset=offset)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3353, in apply_function_on_filtered_inputs
    processed_inputs = function(*fn_args, *additional_args, **fn_kwargs)
  File "/home/yifanhua/5455-term-project/src/main.py", line 256, in <lambda>
    _dataset = _dataset.map(lambda entry: entry['sentence'].str.lower())
AttributeError: 'str' object has no attribute 'str'


Start -- Nov 11 14:17:45 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
14:17:45	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
14:17:47	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
14:29:20	Finish Loading Remote Dataset. Length of dataset: 5
14:29:20	Using Mozilla Common Voice. Additional Cleaning Needed
14:29:20	Starting Size of Mozilla Common Voice: 5
14:30:28	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 410, in <module>
    dataset = load_remote_dataset()
  File "/home/yifanhua/5455-term-project/src/main.py", line 285, in load_remote_dataset
    _dataset = clean_mozilla_dataset(_dataset)
  File "/home/yifanhua/5455-term-project/src/main.py", line 247, in clean_mozilla_dataset
    _dataset = _dataset.filter(lambda entry: True if entry["accent"] else False)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/dataset_dict.py", line 960, in filter
    {
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/dataset_dict.py", line 961, in <dictcomp>
    k: dataset.filter(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 557, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/fingerprint.py", line 511, in wrapper
    out = func(dataset, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3627, in filter
    indices = self.map(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 592, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 557, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3097, in map
    for rank, done, content in Dataset._map_single(**dataset_kwargs):
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3474, in _map_single
    batch = apply_function_on_filtered_inputs(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3353, in apply_function_on_filtered_inputs
    processed_inputs = function(*fn_args, *additional_args, **fn_kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 6236, in get_indices_from_mask_function
    example = {key: batch[key][i] for key in batch}
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 6236, in <dictcomp>
    example = {key: batch[key][i] for key in batch}
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 272, in __getitem__
    value = self.format(key)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 375, in format
    return self.formatter.format_column(self.pa_table.select([key]))
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 442, in format_column
    column = self.python_features_decoder.decode_column(column, pa_table.column_names[0])
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 218, in decode_column
    return self.features.decode_column(column, column_name) if self.features else column
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/features/features.py", line 1925, in decode_column
    [decode_nested_example(self[column_name], value) if value is not None else None for value in column]
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/features/features.py", line 1925, in <listcomp>
    [decode_nested_example(self[column_name], value) if value is not None else None for value in column]
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/features/features.py", line 1325, in decode_nested_example
    return schema.decode_example(obj, token_per_repo_id=token_per_repo_id)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/features/audio.py", line 187, in decode_example
    array, sampling_rate = sf.read(f)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/soundfile.py", line 285, in read
    with SoundFile(file, 'r', samplerate, channels,
  File "/home/yifanhua/.local/lib/python3.10/site-packages/soundfile.py", line 658, in __init__
    self._file = self._open(file, mode_int, closefd)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/soundfile.py", line 1216, in _open
    raise LibsndfileError(err, prefix="Error opening {0!r}: ".format(self.name))
soundfile.LibsndfileError: Error opening <_io.BufferedReader name='/home/yifanhua/.cache/huggingface/datasets/downloads/extracted/a62e68ef77daee3e83f95f22b1da75e786c3ec6a96e2f0e5c5288ea4b46ce202/clips/14dadaeff069012e7c585d95f43a504b52a592ea07ff1871caebd698ca80efbfa8a9bffb448fc93ff976516da5c4ee2841852095cca944ef56dc0abdf107fb0d.mp3'>: Format not recognised.


Start -- Nov 11 14:36:54 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
14:36:54	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
14:37:02	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
14:48:36	Finish Loading Remote Dataset. Length of dataset: 5
14:48:36	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 412, in <module>
    if constants.download_remote_dataset:
  File "/home/yifanhua/5455-term-project/src/main.py", line 286, in load_remote_dataset
    # TODO: Remove next line, we dont need to save dataset now. It's just for speeding up the process of debugging
AttributeError: 'NoneType' object has no attribute 'save_to_disk'


Start -- Nov 11 14:52:16 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
14:52:16	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
14:52:18	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
15:03:55	Finish Loading Remote Dataset. Length of dataset: 12135
15:03:56	Using Mozilla Common Voice. Additional Cleaning Needed
15:03:56	Starting Size of Mozilla Common Voice: 12135
15:04:35	Size after filtering accent: 8671
15:05:01	Size after filtering gender: 8637
15:05:28	Size after filtering voting: 8637
15:05:28	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 413, in <module>
    dataset = load_remote_dataset()
  File "/home/yifanhua/5455-term-project/src/main.py", line 288, in load_remote_dataset
    _dataset = clean_mozilla_dataset(_dataset)
  File "/home/yifanhua/5455-term-project/src/main.py", line 256, in clean_mozilla_dataset
    _dataset = _dataset.map(lambda entry: entry['sentence'].lower())
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 592, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 557, in wrapper
    out: Union["Dataset", "DatasetDict"] = func(self, *args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3097, in map
    for rank, done, content in Dataset._map_single(**dataset_kwargs):
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3450, in _map_single
    example = apply_function_on_filtered_inputs(example, i, offset=offset)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3364, in apply_function_on_filtered_inputs
    validate_function_output(processed_inputs, indices)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 3309, in validate_function_output
    raise TypeError(
TypeError: Provided `function` which is applied to all elements of table returns a variable of type <class 'str'>. Make sure provided `function` returns a variable of type `dict` (or a pyarrow table) to update the dataset or `None` if you are only interested in side effects.


Start -- Nov 11 17:10:50 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
17:10:50	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
17:10:52	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
17:11:01	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 17:11:05 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
17:11:05	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
17:11:08	Using Locally Processed Dataset. Skip Processing...
17:11:08	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
17:11:09	Input text: I'm loading the model from the Hugging Face Hub!
17:11:09	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 440, in <module>
    speaker_embeddings = torch.tensor(example["speaker_embeddings"]).unsqueeze(0).to(device)
KeyError: 'speaker_embeddings'


Start -- Nov 11 17:11:49 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
17:11:49	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
17:11:52	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
17:23:25	Finish Loading Remote Dataset. Length of dataset: 12135
17:23:26	Using Mozilla Common Voice. Additional Cleaning Needed
17:23:26	Starting Size of Mozilla Common Voice: 12135
17:24:05	Size after filtering accent: 8671
17:24:32	Size after filtering gender: 8637
17:24:58	Size after filtering voting: 8637
17:24:59	Finish normalizing input text
17:24:59	Finish Cleaning Dataset. Length of dataset: 8637
17:24:59	Start Setting Sampling Rate to 16 kHz...
17:24:59	Setting Sampling Rate Successfully
17:24:59	Start Filtering Short Data
17:24:59	8635 data entries left after filtering
17:24:59	Start Preparing Dataset
17:27:40	Preparing dataset finished successfully
17:27:40	Start Filtering Long Data
17:27:40	8635 data entries left after filtering
17:27:40	save_processed_dataset is True. Saving processed dataset to dir: ./data/
17:27:41	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
17:27:42	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 11 19:11:38 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
19:11:38	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
19:11:40	Using Locally Processed Dataset. Skip Processing...
19:11:40	Using model: ./model/trained_yifanhua-threadripper_Nov10_21:39
19:11:41	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 11 19:21:50 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
19:21:50	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
19:21:53	Using Locally Processed Dataset. Skip Processing...
19:21:53	Generating Seq2SeqTrainer Arguments
19:21:53	Generating Seq2SeqTrainer
19:21:53	Start Training...
19:44:36	Start Saving the model...
19:44:41	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 11 19:57:56 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
19:57:56	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
19:57:59	Using Locally Processed Dataset. Skip Processing...
19:57:59	Generating Seq2SeqTrainer Arguments
19:57:59	Generating Seq2SeqTrainer
19:57:59	Start Training...
19:58:24	KeyboardInterrupt detected. Exiting...

Start -- Nov 11 19:58:37 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
19:58:37	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
19:58:39	Using Locally Processed Dataset. Skip Processing...
19:58:39	Generating Seq2SeqTrainer Arguments
19:58:39	Generating Seq2SeqTrainer
19:58:39	Start Training...
20:26:44	Start Saving the model...
20:26:49	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 13 15:54:00 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
15:54:00	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
15:54:03	Using Locally Processed Dataset. Skip Processing...
15:54:03	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 415, in <module>
    sort_speaker(dataset)
  File "/home/yifanhua/5455-term-project/src/main.py", line 313, in sort_speaker
    for speaker_id in _dataset["speaker_id"]:
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 2803, in __getitem__
    return self._getitem(key)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 2787, in _getitem
    pa_subtable = query_table(self._data, key, indices=self._indices if self._indices is not None else None)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 580, in query_table
    _check_valid_column_key(key, table.column_names)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 520, in _check_valid_column_key
    raise KeyError(f"Column {key} not in the dataset. Current columns in the dataset: {columns}")
KeyError: "Column speaker_id not in the dataset. Current columns in the dataset: ['input_ids', 'labels', 'speaker_embeddings']"


Start -- Nov 13 15:55:06 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
15:55:06	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
15:55:08	Using Locally Processed Dataset. Skip Processing...
15:55:08	Generating Seq2SeqTrainer Arguments
15:55:08	Generating Seq2SeqTrainer
15:55:08	Start Training...
15:55:33	KeyboardInterrupt detected. Exiting...

Start -- Nov 13 15:55:40 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
15:55:40	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
15:55:42	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
16:07:36	Finish Loading Remote Dataset. Length of dataset: 12135
16:07:36	Using Mozilla Common Voice. Additional Cleaning Needed
16:07:36	Starting Size of Mozilla Common Voice: 12135
16:08:18	Size after filtering accent: 8671
16:08:45	Size after filtering gender: 8637
16:09:12	Size after filtering voting: 8637
16:09:13	Finish normalizing input text
16:09:13	Finish Cleaning Dataset. Length of dataset: 8637
16:09:13	Start Setting Sampling Rate to 16 kHz...
16:09:13	Setting Sampling Rate Successfully
16:09:13	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 406, in <module>
    sort_speaker(dataset)
  File "/home/yifanhua/5455-term-project/src/main.py", line 313, in sort_speaker
    for speaker_id in _dataset["speaker_id"]:
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 2803, in __getitem__
    return self._getitem(key)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/arrow_dataset.py", line 2787, in _getitem
    pa_subtable = query_table(self._data, key, indices=self._indices if self._indices is not None else None)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 580, in query_table
    _check_valid_column_key(key, table.column_names)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/formatting/formatting.py", line 520, in _check_valid_column_key
    raise KeyError(f"Column {key} not in the dataset. Current columns in the dataset: {columns}")
KeyError: "Column speaker_id not in the dataset. Current columns in the dataset: ['client_id', 'path', 'audio', 'normalized_text', 'up_votes', 'down_votes', 'age', 'gender', 'accent', 'locale', 'segment']"


Start -- Nov 13 18:47:37 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
18:47:37	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
18:47:39	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
18:59:17	Finish Loading Remote Dataset. Length of dataset: 12135
18:59:17	Using Mozilla Common Voice. Additional Cleaning Needed
18:59:17	Starting Size of Mozilla Common Voice: 12135
18:59:57	Size after filtering accent: 8671
19:00:24	Size after filtering gender: 8637
19:00:50	Size after filtering voting: 8637
19:00:51	Finish normalizing input text
19:00:51	Finish Cleaning Dataset. Length of dataset: 8637
19:00:51	Start Setting Sampling Rate to 16 kHz...
19:00:51	Setting Sampling Rate Successfully
19:00:52	Start Filtering Short Data
19:00:52	8635 data entries left after filtering
19:00:52	Start Preparing Dataset
19:03:39	Preparing dataset finished successfully
19:03:39	Start Filtering Long Data
19:03:39	8635 data entries left after filtering
19:03:39	save_processed_dataset is True. Saving processed dataset to dir: ./data/
19:03:40	Generating Seq2SeqTrainer Arguments
19:03:40	Generating Seq2SeqTrainer
19:03:41	Start Training...
19:19:45	KeyboardInterrupt detected. Exiting...

Start -- Nov 13 19:19:50 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
19:19:50	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
19:19:52	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
19:20:04	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 439, in <module>
    dataset = load_remote_dataset()
  File "/home/yifanhua/5455-term-project/src/main.py", line 272, in load_remote_dataset
    _dataset = load_dataset(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/load.py", line 2153, in load_dataset
    builder_instance.download_and_prepare(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/builder.py", line 954, in download_and_prepare
    self._download_and_prepare(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/builder.py", line 1717, in _download_and_prepare
    super()._download_and_prepare(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/datasets/builder.py", line 1027, in _download_and_prepare
    split_generators = self._split_generators(dl_manager, **split_generators_kwargs)
  File "/home/yifanhua/.cache/huggingface/modules/datasets_modules/datasets/mozilla-foundation--common_voice_1_0/f01470c32b0b93cfcd4baaa7725f5ec0285549a5d086fab27151c3260e5c8035/common_voice_1_0.py", line 159, in _split_generators
    archive_path = dl_manager.download(self._get_bundle_url(self.config.name, bundle_url_template))
  File "/home/yifanhua/.cache/huggingface/modules/datasets_modules/datasets/mozilla-foundation--common_voice_1_0/f01470c32b0b93cfcd4baaa7725f5ec0285549a5d086fab27151c3260e5c8035/common_voice_1_0.py", line 135, in _get_bundle_url
    response = requests.get(f"{_API_URL}/bucket/dataset/{path}", timeout=10.0).json()
  File "/usr/lib/python3/dist-packages/requests/models.py", line 900, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 2 column 1 (char 1)


Start -- Nov 13 19:26:45 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
19:26:45	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
19:26:47	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
19:38:23	Finish Loading Remote Dataset. Length of dataset: 12135
19:38:23	Using Mozilla Common Voice. Additional Cleaning Needed
19:38:23	Starting Size of Mozilla Common Voice: 12135
19:39:02	Size after filtering accent: 8671
19:39:29	Size after filtering gender: 8637
19:39:55	Size after filtering voting: 8637
19:39:56	Finish normalizing input text
19:39:56	Finish Cleaning Dataset. Length of dataset: 8637
19:39:56	Start Setting Sampling Rate to 16 kHz...
19:39:56	Setting Sampling Rate Successfully
19:39:57	Start Filtering Short Data
19:39:57	8635 data entries left after filtering
19:39:57	Start Preparing Dataset
19:42:42	Preparing dataset finished successfully
19:42:42	Start Filtering Long Data
19:42:42	8635 data entries left after filtering
19:42:42	save_processed_dataset is True. Saving processed dataset to dir: ./data/
19:42:44	Generating Seq2SeqTrainer Arguments
19:42:44	Generating Seq2SeqTrainer
19:42:44	Start Training...
20:18:29	Start Saving the model...
20:18:34	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 14 09:13:00 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
09:13:00	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
09:13:05	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
09:24:58	Finish Loading Remote Dataset. Length of dataset: 12135
09:24:58	Using Mozilla Common Voice. Additional Cleaning Needed
09:24:58	Starting Size of Mozilla Common Voice: 12135
09:25:40	Size after filtering accent: 8671
09:26:07	Size after filtering gender: 8637
09:26:34	Size after filtering voting: 8637
09:26:35	Finish normalizing input text
09:26:35	Finish Cleaning Dataset. Length of dataset: 8637
09:26:35	Start Setting Sampling Rate to 16 kHz...
09:26:35	Setting Sampling Rate Successfully
09:26:36	Start Filtering Short Data
09:26:36	8635 data entries left after filtering
09:26:36	Start Preparing Dataset
09:29:20	Preparing dataset finished successfully
09:29:20	Start Filtering Long Data
09:29:20	8635 data entries left after filtering
09:29:20	save_processed_dataset is True. Saving processed dataset to dir: ./data/
09:29:21	Generating Seq2SeqTrainer Arguments
09:29:21	Generating Seq2SeqTrainer
09:29:21	Exception: Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 169, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3/dist-packages/urllib3/util/connection.py", line 73, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 700, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 383, in _make_request
    self._validate_conn(conn)
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 1017, in _validate_conn
    conn.connect()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 353, in connect
    conn = self._new_conn()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 181, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f77eb955060>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 756, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3/dist-packages/urllib3/util/retry.py", line 574, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='huggingface.co', port=443): Max retries exceeded with url: /api/repos/create (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f77eb955060>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 461, in <module>
    trainer = generate_trainer(training_args, pretrained_model, divided_dataset, data_collator, tokenizer)
  File "/home/yifanhua/5455-term-project/src/main.py", line 425, in generate_trainer
    return Seq2SeqTrainer(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/trainer_seq2seq.py", line 56, in __init__
    super().__init__(
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/trainer.py", line 536, in __init__
    self.init_hf_repo()
  File "/home/yifanhua/.local/lib/python3.10/site-packages/transformers/trainer.py", line 3480, in init_hf_repo
    repo_url = create_repo(repo_name, token=self.args.hub_token, private=self.args.hub_private_repo, exist_ok=True)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/huggingface_hub/utils/_validators.py", line 118, in _inner_fn
    return fn(*args, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/huggingface_hub/hf_api.py", line 3053, in create_repo
    r = get_session().post(path, headers=headers, json=json)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 592, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 544, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 657, in send
    r = adapter.send(request, **kwargs)
  File "/home/yifanhua/.local/lib/python3.10/site-packages/huggingface_hub/utils/_http.py", line 63, in send
    return super().send(request, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: (MaxRetryError("HTTPSConnectionPool(host='huggingface.co', port=443): Max retries exceeded with url: /api/repos/create (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f77eb955060>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"), '(Request ID: 066c2541-fa49-48d2-8041-8d560bde5683)')


Start -- Nov 14 10:10:48 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:10:48	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
10:10:50	Using Locally Processed Dataset. Skip Processing...
10:10:50	Generating Seq2SeqTrainer Arguments
10:10:50	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 467, in <module>
    training_args = generate_train_arguments()
  File "/home/yifanhua/5455-term-project/src/main.py", line 406, in generate_train_arguments
    return Seq2SeqTrainingArguments(
TypeError: Seq2SeqTrainingArguments.__init__() got an unexpected keyword argument 'log'


Start -- Nov 14 10:11:02 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:11:02	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
10:11:04	Using Locally Processed Dataset. Skip Processing...
10:11:04	Generating Seq2SeqTrainer Arguments
10:11:04	Generating Seq2SeqTrainer
10:11:04	Start Training...
10:14:38	KeyboardInterrupt detected. Exiting...

Start -- Nov 14 10:18:54 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:18:54	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
10:18:56	Using Locally Processed Dataset. Skip Processing...
10:18:56	Generating Seq2SeqTrainer Arguments
10:18:56	Generating Seq2SeqTrainer
10:18:56	Start Training...
10:25:02	KeyboardInterrupt detected. Exiting...

Start -- Nov 14 10:25:05 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:25:05	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
10:25:07	Using Locally Processed Dataset. Skip Processing...
10:25:07	Generating Seq2SeqTrainer Arguments
10:25:07	Generating Seq2SeqTrainer
10:25:07	Start Training...
10:25:15	KeyboardInterrupt detected. Exiting...

Start -- Nov 14 10:25:19 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:25:19	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
10:25:21	Using Locally Processed Dataset. Skip Processing...
10:25:21	Generating Seq2SeqTrainer Arguments
10:25:21	Generating Seq2SeqTrainer
10:25:21	Start Training...
10:26:20	KeyboardInterrupt detected. Exiting...

Start -- Nov 14 10:26:24 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:26:24	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
loading configuration file config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/config.json
Model config SpeechT5Config {
  "activation_dropout": 0.1,
  "apply_spec_augment": true,
  "architectures": [
    "SpeechT5ForTextToSpeech"
  ],
  "attention_dropout": 0.1,
  "bos_token_id": 0,
  "conv_bias": false,
  "conv_dim": [
    512,
    512,
    512,
    512,
    512,
    512,
    512
  ],
  "conv_kernel": [
    10,
    3,
    3,
    3,
    3,
    2,
    2
  ],
  "conv_stride": [
    5,
    2,
    2,
    2,
    2,
    2,
    2
  ],
  "decoder_attention_heads": 12,
  "decoder_ffn_dim": 3072,
  "decoder_layerdrop": 0.1,
  "decoder_layers": 6,
  "decoder_start_token_id": 2,
  "encoder_attention_heads": 12,
  "encoder_ffn_dim": 3072,
  "encoder_layerdrop": 0.1,
  "encoder_layers": 12,
  "encoder_max_relative_position": 160,
  "eos_token_id": 2,
  "feat_extract_activation": "gelu",
  "feat_extract_norm": "group",
  "feat_proj_dropout": 0.0,
  "guided_attention_loss_num_heads": 2,
  "guided_attention_loss_scale": 10.0,
  "guided_attention_loss_sigma": 0.4,
  "hidden_act": "gelu",
  "hidden_dropout": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "is_encoder_decoder": true,
  "layer_norm_eps": 1e-05,
  "mask_feature_length": 10,
  "mask_feature_min_masks": 0,
  "mask_feature_prob": 0.0,
  "mask_time_length": 10,
  "mask_time_min_masks": 2,
  "mask_time_prob": 0.05,
  "max_length": 1876,
  "max_speech_positions": 1876,
  "max_text_positions": 600,
  "model_type": "speecht5",
  "num_conv_pos_embedding_groups": 16,
  "num_conv_pos_embeddings": 128,
  "num_feat_extract_layers": 7,
  "num_mel_bins": 80,
  "pad_token_id": 1,
  "positional_dropout": 0.1,
  "reduction_factor": 2,
  "scale_embedding": false,
  "speaker_embedding_dim": 512,
  "speech_decoder_postnet_dropout": 0.5,
  "speech_decoder_postnet_kernel": 5,
  "speech_decoder_postnet_layers": 5,
  "speech_decoder_postnet_units": 256,
  "speech_decoder_prenet_dropout": 0.5,
  "speech_decoder_prenet_layers": 2,
  "speech_decoder_prenet_units": 256,
  "torch_dtype": "float32",
  "transformers_version": "4.36.0.dev0",
  "use_cache": true,
  "use_guided_attention_loss": true,
  "vocab_size": 81
}

loading weights file pytorch_model.bin from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/pytorch_model.bin
Generate config GenerationConfig {
  "bos_token_id": 0,
  "decoder_start_token_id": 2,
  "eos_token_id": 2,
  "max_length": 1876,
  "pad_token_id": 1
}

All model checkpoint weights were used when initializing SpeechT5ForTextToSpeech.

All the weights of SpeechT5ForTextToSpeech were initialized from the model checkpoint at microsoft/speecht5_tts.
If your task is similar to the task the model of the checkpoint was trained on, you can already use SpeechT5ForTextToSpeech for predictions without further training.
Generation config file not found, using a generation config created from the model config.
loading configuration file preprocessor_config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/preprocessor_config.json
Feature extractor SpeechT5FeatureExtractor {
  "do_normalize": false,
  "feature_extractor_type": "SpeechT5FeatureExtractor",
  "feature_size": 1,
  "fmax": 7600,
  "fmin": 80,
  "frame_signal_scale": 1.0,
  "hop_length": 16,
  "mel_floor": 1e-10,
  "num_mel_bins": 80,
  "padding_side": "right",
  "padding_value": 0.0,
  "processor_class": "SpeechT5Processor",
  "reduction_factor": 2,
  "return_attention_mask": true,
  "sampling_rate": 16000,
  "win_function": "hann_window",
  "win_length": 64
}

loading file spm_char.model from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/spm_char.model
loading file added_tokens.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/added_tokens.json
loading file special_tokens_map.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/special_tokens_map.json
loading file tokenizer_config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/tokenizer_config.json
loading file tokenizer.json from cache at None
loading configuration file config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_hifigan/snapshots/bb6f429406e86a9992357a972c0698b22043307d/config.json
Model config SpeechT5HifiGanConfig {
  "architectures": [
    "SpeechT5HifiGan"
  ],
  "initializer_range": 0.01,
  "leaky_relu_slope": 0.1,
  "model_in_dim": 80,
  "model_type": "hifigan",
  "normalize_before": true,
  "resblock_dilation_sizes": [
    [
      1,
      3,
      5
    ],
    [
      1,
      3,
      5
    ],
    [
      1,
      3,
      5
    ]
  ],
  "resblock_kernel_sizes": [
    3,
    7,
    11
  ],
  "sampling_rate": 16000,
  "torch_dtype": "float32",
  "transformers_version": "4.36.0.dev0",
  "upsample_initial_channel": 512,
  "upsample_kernel_sizes": [
    8,
    8,
    8,
    8
  ],
  "upsample_rates": [
    4,
    4,
    4,
    4
  ]
}

loading weights file pytorch_model.bin from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_hifigan/snapshots/bb6f429406e86a9992357a972c0698b22043307d/pytorch_model.bin
All model checkpoint weights were used when initializing SpeechT5HifiGan.

All the weights of SpeechT5HifiGan were initialized from the model checkpoint at microsoft/speecht5_hifigan.
If your task is similar to the task the model of the checkpoint was trained on, you can already use SpeechT5HifiGan for predictions without further training.
10:26:26	Using Locally Processed Dataset. Skip Processing...
10:26:26	Generating Seq2SeqTrainer Arguments
PyTorch: setting up devices
10:26:26	Generating Seq2SeqTrainer
max_steps is given, it will override any value given in num_train_epochs
Using auto half precision backend
10:26:26	Start Training...
Currently training with a batch size of: 8
***** Running training *****
  Num examples = 2,000
  Num Epochs = 32
  Instantaneous batch size per device = 8
  Total train batch size (w. parallel, distributed & accumulation) = 16
  Gradient Accumulation steps = 2
  Total optimization steps = 4,000
  Number of trainable parameters = 144,431,684
10:27:11	KeyboardInterrupt detected. Exiting...

Start -- Nov 14 10:27:14 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:27:14	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
loading configuration file config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/config.json
Model config SpeechT5Config {
  "activation_dropout": 0.1,
  "apply_spec_augment": true,
  "architectures": [
    "SpeechT5ForTextToSpeech"
  ],
  "attention_dropout": 0.1,
  "bos_token_id": 0,
  "conv_bias": false,
  "conv_dim": [
    512,
    512,
    512,
    512,
    512,
    512,
    512
  ],
  "conv_kernel": [
    10,
    3,
    3,
    3,
    3,
    2,
    2
  ],
  "conv_stride": [
    5,
    2,
    2,
    2,
    2,
    2,
    2
  ],
  "decoder_attention_heads": 12,
  "decoder_ffn_dim": 3072,
  "decoder_layerdrop": 0.1,
  "decoder_layers": 6,
  "decoder_start_token_id": 2,
  "encoder_attention_heads": 12,
  "encoder_ffn_dim": 3072,
  "encoder_layerdrop": 0.1,
  "encoder_layers": 12,
  "encoder_max_relative_position": 160,
  "eos_token_id": 2,
  "feat_extract_activation": "gelu",
  "feat_extract_norm": "group",
  "feat_proj_dropout": 0.0,
  "guided_attention_loss_num_heads": 2,
  "guided_attention_loss_scale": 10.0,
  "guided_attention_loss_sigma": 0.4,
  "hidden_act": "gelu",
  "hidden_dropout": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "is_encoder_decoder": true,
  "layer_norm_eps": 1e-05,
  "mask_feature_length": 10,
  "mask_feature_min_masks": 0,
  "mask_feature_prob": 0.0,
  "mask_time_length": 10,
  "mask_time_min_masks": 2,
  "mask_time_prob": 0.05,
  "max_length": 1876,
  "max_speech_positions": 1876,
  "max_text_positions": 600,
  "model_type": "speecht5",
  "num_conv_pos_embedding_groups": 16,
  "num_conv_pos_embeddings": 128,
  "num_feat_extract_layers": 7,
  "num_mel_bins": 80,
  "pad_token_id": 1,
  "positional_dropout": 0.1,
  "reduction_factor": 2,
  "scale_embedding": false,
  "speaker_embedding_dim": 512,
  "speech_decoder_postnet_dropout": 0.5,
  "speech_decoder_postnet_kernel": 5,
  "speech_decoder_postnet_layers": 5,
  "speech_decoder_postnet_units": 256,
  "speech_decoder_prenet_dropout": 0.5,
  "speech_decoder_prenet_layers": 2,
  "speech_decoder_prenet_units": 256,
  "torch_dtype": "float32",
  "transformers_version": "4.36.0.dev0",
  "use_cache": true,
  "use_guided_attention_loss": true,
  "vocab_size": 81
}

loading weights file pytorch_model.bin from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/pytorch_model.bin
Generate config GenerationConfig {
  "bos_token_id": 0,
  "decoder_start_token_id": 2,
  "eos_token_id": 2,
  "max_length": 1876,
  "pad_token_id": 1
}

All model checkpoint weights were used when initializing SpeechT5ForTextToSpeech.

All the weights of SpeechT5ForTextToSpeech were initialized from the model checkpoint at microsoft/speecht5_tts.
If your task is similar to the task the model of the checkpoint was trained on, you can already use SpeechT5ForTextToSpeech for predictions without further training.
Generation config file not found, using a generation config created from the model config.
loading configuration file preprocessor_config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/preprocessor_config.json
Feature extractor SpeechT5FeatureExtractor {
  "do_normalize": false,
  "feature_extractor_type": "SpeechT5FeatureExtractor",
  "feature_size": 1,
  "fmax": 7600,
  "fmin": 80,
  "frame_signal_scale": 1.0,
  "hop_length": 16,
  "mel_floor": 1e-10,
  "num_mel_bins": 80,
  "padding_side": "right",
  "padding_value": 0.0,
  "processor_class": "SpeechT5Processor",
  "reduction_factor": 2,
  "return_attention_mask": true,
  "sampling_rate": 16000,
  "win_function": "hann_window",
  "win_length": 64
}

loading file spm_char.model from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/spm_char.model
loading file added_tokens.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/added_tokens.json
loading file special_tokens_map.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/special_tokens_map.json
loading file tokenizer_config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/tokenizer_config.json
loading file tokenizer.json from cache at None
loading configuration file config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_hifigan/snapshots/bb6f429406e86a9992357a972c0698b22043307d/config.json
Model config SpeechT5HifiGanConfig {
  "architectures": [
    "SpeechT5HifiGan"
  ],
  "initializer_range": 0.01,
  "leaky_relu_slope": 0.1,
  "model_in_dim": 80,
  "model_type": "hifigan",
  "normalize_before": true,
  "resblock_dilation_sizes": [
    [
      1,
      3,
      5
    ],
    [
      1,
      3,
      5
    ],
    [
      1,
      3,
      5
    ]
  ],
  "resblock_kernel_sizes": [
    3,
    7,
    11
  ],
  "sampling_rate": 16000,
  "torch_dtype": "float32",
  "transformers_version": "4.36.0.dev0",
  "upsample_initial_channel": 512,
  "upsample_kernel_sizes": [
    8,
    8,
    8,
    8
  ],
  "upsample_rates": [
    4,
    4,
    4,
    4
  ]
}

loading weights file pytorch_model.bin from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_hifigan/snapshots/bb6f429406e86a9992357a972c0698b22043307d/pytorch_model.bin
All model checkpoint weights were used when initializing SpeechT5HifiGan.

All the weights of SpeechT5HifiGan were initialized from the model checkpoint at microsoft/speecht5_hifigan.
If your task is similar to the task the model of the checkpoint was trained on, you can already use SpeechT5HifiGan for predictions without further training.
10:27:17	Using Locally Processed Dataset. Skip Processing...
10:27:17	Generating Seq2SeqTrainer Arguments
PyTorch: setting up devices
10:27:17	Generating Seq2SeqTrainer
max_steps is given, it will override any value given in num_train_epochs
Using auto half precision backend
10:27:17	Start Training...
Currently training with a batch size of: 8
***** Running training *****
  Num examples = 2,000
  Num Epochs = 1
  Instantaneous batch size per device = 8
  Total train batch size (w. parallel, distributed & accumulation) = 16
  Gradient Accumulation steps = 2
  Total optimization steps = 100
  Number of trainable parameters = 144,431,684
***** Running Evaluation *****
  Num examples = 5
  Batch size = 8
Saving model checkpoint to ./speecht5_tts/checkpoint-100
Configuration saved in ./speecht5_tts/checkpoint-100/config.json
Configuration saved in ./speecht5_tts/checkpoint-100/generation_config.json
Model weights saved in ./speecht5_tts/checkpoint-100/pytorch_model.bin
tokenizer config file saved in ./speecht5_tts/checkpoint-100/tokenizer_config.json
Special tokens file saved in ./speecht5_tts/checkpoint-100/special_tokens_map.json
added tokens file saved in ./speecht5_tts/checkpoint-100/added_tokens.json
tokenizer config file saved in ./speecht5_tts/tokenizer_config.json
Special tokens file saved in ./speecht5_tts/special_tokens_map.json
added tokens file saved in ./speecht5_tts/added_tokens.json


Training completed. Do not forget to share your model on huggingface.co/models =)


Loading best model from ./speecht5_tts/checkpoint-100 (score: 0.6336874961853027).
Waiting for the current checkpoint push to be finished, this might take a couple of minutes.
10:28:32	Start Saving the model...
Saving model checkpoint to ./model/trained_yifanhua-threadripper_Nov14_10:27
Configuration saved in ./model/trained_yifanhua-threadripper_Nov14_10:27/config.json
Configuration saved in ./model/trained_yifanhua-threadripper_Nov14_10:27/generation_config.json
Model weights saved in ./model/trained_yifanhua-threadripper_Nov14_10:27/pytorch_model.bin
tokenizer config file saved in ./model/trained_yifanhua-threadripper_Nov14_10:27/tokenizer_config.json
Special tokens file saved in ./model/trained_yifanhua-threadripper_Nov14_10:27/special_tokens_map.json
added tokens file saved in ./model/trained_yifanhua-threadripper_Nov14_10:27/added_tokens.json
Saving model checkpoint to ./speecht5_tts/
Configuration saved in ./speecht5_tts/config.json
Configuration saved in ./speecht5_tts/generation_config.json
Model weights saved in ./speecht5_tts/pytorch_model.bin
tokenizer config file saved in ./speecht5_tts/tokenizer_config.json
Special tokens file saved in ./speecht5_tts/special_tokens_map.json
added tokens file saved in ./speecht5_tts/added_tokens.json
Dropping the following result as it does not have all the necessary fields:
{'dataset': {'name': 'common_voice_1_0', 'type': 'common_voice_1_0', 'config': 'en', 'split': 'train', 'args': 'en'}}
Saving model checkpoint to ./speecht5_tts/
Configuration saved in ./speecht5_tts/config.json
Configuration saved in ./speecht5_tts/generation_config.json
Model weights saved in ./speecht5_tts/pytorch_model.bin
tokenizer config file saved in ./speecht5_tts/tokenizer_config.json
Special tokens file saved in ./speecht5_tts/special_tokens_map.json
added tokens file saved in ./speecht5_tts/added_tokens.json
Dropping the following result as it does not have all the necessary fields:
{'dataset': {'name': 'Common Voice', 'type': 'mozilla-foundation/common_voice_1_0', 'config': 'en', 'split': 'train', 'args': 'config: en'}}
10:28:38	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 14 10:32:36 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:32:36	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
loading configuration file config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/config.json
Model config SpeechT5Config {
  "activation_dropout": 0.1,
  "apply_spec_augment": true,
  "architectures": [
    "SpeechT5ForTextToSpeech"
  ],
  "attention_dropout": 0.1,
  "bos_token_id": 0,
  "conv_bias": false,
  "conv_dim": [
    512,
    512,
    512,
    512,
    512,
    512,
    512
  ],
  "conv_kernel": [
    10,
    3,
    3,
    3,
    3,
    2,
    2
  ],
  "conv_stride": [
    5,
    2,
    2,
    2,
    2,
    2,
    2
  ],
  "decoder_attention_heads": 12,
  "decoder_ffn_dim": 3072,
  "decoder_layerdrop": 0.1,
  "decoder_layers": 6,
  "decoder_start_token_id": 2,
  "encoder_attention_heads": 12,
  "encoder_ffn_dim": 3072,
  "encoder_layerdrop": 0.1,
  "encoder_layers": 12,
  "encoder_max_relative_position": 160,
  "eos_token_id": 2,
  "feat_extract_activation": "gelu",
  "feat_extract_norm": "group",
  "feat_proj_dropout": 0.0,
  "guided_attention_loss_num_heads": 2,
  "guided_attention_loss_scale": 10.0,
  "guided_attention_loss_sigma": 0.4,
  "hidden_act": "gelu",
  "hidden_dropout": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "is_encoder_decoder": true,
  "layer_norm_eps": 1e-05,
  "mask_feature_length": 10,
  "mask_feature_min_masks": 0,
  "mask_feature_prob": 0.0,
  "mask_time_length": 10,
  "mask_time_min_masks": 2,
  "mask_time_prob": 0.05,
  "max_length": 1876,
  "max_speech_positions": 1876,
  "max_text_positions": 600,
  "model_type": "speecht5",
  "num_conv_pos_embedding_groups": 16,
  "num_conv_pos_embeddings": 128,
  "num_feat_extract_layers": 7,
  "num_mel_bins": 80,
  "pad_token_id": 1,
  "positional_dropout": 0.1,
  "reduction_factor": 2,
  "scale_embedding": false,
  "speaker_embedding_dim": 512,
  "speech_decoder_postnet_dropout": 0.5,
  "speech_decoder_postnet_kernel": 5,
  "speech_decoder_postnet_layers": 5,
  "speech_decoder_postnet_units": 256,
  "speech_decoder_prenet_dropout": 0.5,
  "speech_decoder_prenet_layers": 2,
  "speech_decoder_prenet_units": 256,
  "torch_dtype": "float32",
  "transformers_version": "4.36.0.dev0",
  "use_cache": true,
  "use_guided_attention_loss": true,
  "vocab_size": 81
}

loading weights file pytorch_model.bin from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/pytorch_model.bin
Generate config GenerationConfig {
  "bos_token_id": 0,
  "decoder_start_token_id": 2,
  "eos_token_id": 2,
  "max_length": 1876,
  "pad_token_id": 1
}

All model checkpoint weights were used when initializing SpeechT5ForTextToSpeech.

All the weights of SpeechT5ForTextToSpeech were initialized from the model checkpoint at microsoft/speecht5_tts.
If your task is similar to the task the model of the checkpoint was trained on, you can already use SpeechT5ForTextToSpeech for predictions without further training.
Generation config file not found, using a generation config created from the model config.
loading configuration file preprocessor_config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/preprocessor_config.json
Feature extractor SpeechT5FeatureExtractor {
  "do_normalize": false,
  "feature_extractor_type": "SpeechT5FeatureExtractor",
  "feature_size": 1,
  "fmax": 7600,
  "fmin": 80,
  "frame_signal_scale": 1.0,
  "hop_length": 16,
  "mel_floor": 1e-10,
  "num_mel_bins": 80,
  "padding_side": "right",
  "padding_value": 0.0,
  "processor_class": "SpeechT5Processor",
  "reduction_factor": 2,
  "return_attention_mask": true,
  "sampling_rate": 16000,
  "win_function": "hann_window",
  "win_length": 64
}

loading file spm_char.model from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/spm_char.model
loading file added_tokens.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/added_tokens.json
loading file special_tokens_map.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/special_tokens_map.json
loading file tokenizer_config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_tts/snapshots/30fcde30f19b87502b8435427b5f5068e401d5f6/tokenizer_config.json
loading file tokenizer.json from cache at None
loading configuration file config.json from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_hifigan/snapshots/bb6f429406e86a9992357a972c0698b22043307d/config.json
Model config SpeechT5HifiGanConfig {
  "architectures": [
    "SpeechT5HifiGan"
  ],
  "initializer_range": 0.01,
  "leaky_relu_slope": 0.1,
  "model_in_dim": 80,
  "model_type": "hifigan",
  "normalize_before": true,
  "resblock_dilation_sizes": [
    [
      1,
      3,
      5
    ],
    [
      1,
      3,
      5
    ],
    [
      1,
      3,
      5
    ]
  ],
  "resblock_kernel_sizes": [
    3,
    7,
    11
  ],
  "sampling_rate": 16000,
  "torch_dtype": "float32",
  "transformers_version": "4.36.0.dev0",
  "upsample_initial_channel": 512,
  "upsample_kernel_sizes": [
    8,
    8,
    8,
    8
  ],
  "upsample_rates": [
    4,
    4,
    4,
    4
  ]
}

loading weights file pytorch_model.bin from cache at /home/yifanhua/.cache/huggingface/hub/models--microsoft--speecht5_hifigan/snapshots/bb6f429406e86a9992357a972c0698b22043307d/pytorch_model.bin
All model checkpoint weights were used when initializing SpeechT5HifiGan.

All the weights of SpeechT5HifiGan were initialized from the model checkpoint at microsoft/speecht5_hifigan.
If your task is similar to the task the model of the checkpoint was trained on, you can already use SpeechT5HifiGan for predictions without further training.
10:32:38	Using Locally Processed Dataset. Skip Processing...
10:32:38	Generating Seq2SeqTrainer Arguments
10:32:38	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 468, in <module>
    training_args = generate_train_arguments()
  File "/home/yifanhua/5455-term-project/src/main.py", line 408, in generate_train_arguments
    return Seq2SeqTrainingArguments(
TypeError: Seq2SeqTrainingArguments.__init__() got an unexpected keyword argument 'callbacks'


Start -- Nov 14 10:56:17 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
10:56:17	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
10:56:19	Using Locally Processed Dataset. Skip Processing...
10:56:19	Generating Seq2SeqTrainer Arguments
10:56:19	Generating Seq2SeqTrainer
10:56:20	Start Training...
10:57:33	Start Saving the model...
10:57:38	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 14 12:24:04 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
12:24:04	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
12:24:07	Using Locally Processed Dataset. Skip Processing...
12:24:07	Generating Seq2SeqTrainer Arguments
12:24:07	Generating Seq2SeqTrainer
12:24:07	Start Training...
12:25:16	Start Saving the model...
12:25:21	Input text: I'm loading the model from the Hugging Face Hub!


Start -- Nov 14 12:30:50 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
12:30:50	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
12:30:53	Using Locally Processed Dataset. Skip Processing...
12:30:53	Generating Seq2SeqTrainer Arguments
12:30:53	Generating Seq2SeqTrainer
12:30:53	Start Training...
12:32:04	Exception: Traceback (most recent call last):
  File "/home/yifanhua/5455-term-project/src/main.py", line 471, in <module>
    log_msg(trainer.state.log_history)
  File "/home/yifanhua/5455-term-project/src/main.py", line 32, in log_msg
    msg = time.strftime("%H:%M:%S", time.localtime()) + '\t' + m if include_time else m
TypeError: can only concatenate str (not "dict") to str


Start -- Nov 14 12:33:09 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
12:33:09	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
12:33:12	Using Locally Processed Dataset. Skip Processing...
12:33:12	Generating Seq2SeqTrainer Arguments
12:33:12	Generating Seq2SeqTrainer
12:33:12	Start Training...
12:34:28	{'loss': 0.9859, 'learning_rate': 9.600000000000001e-06, 'epoch': 0.04, 'step': 5}
12:34:28	{'loss': 0.8889, 'learning_rate': 9.100000000000001e-06, 'epoch': 0.08, 'step': 10}
12:34:28	{'loss': 0.9105, 'learning_rate': 8.6e-06, 'epoch': 0.12, 'step': 15}
12:34:28	{'loss': 0.8511, 'learning_rate': 8.1e-06, 'epoch': 0.16, 'step': 20}
12:34:28	{'loss': 0.8137, 'learning_rate': 7.7e-06, 'epoch': 0.2, 'step': 25}
12:34:28	{'loss': 0.8476, 'learning_rate': 7.2000000000000005e-06, 'epoch': 0.24, 'step': 30}
12:34:28	{'loss': 0.8663, 'learning_rate': 6.700000000000001e-06, 'epoch': 0.28, 'step': 35}
12:34:28	{'loss': 0.8475, 'learning_rate': 6.200000000000001e-06, 'epoch': 0.32, 'step': 40}
12:34:28	{'loss': 0.8243, 'learning_rate': 5.7e-06, 'epoch': 0.36, 'step': 45}
12:34:28	{'loss': 0.8349, 'learning_rate': 5.2e-06, 'epoch': 0.4, 'step': 50}
12:34:28	{'loss': 0.7796, 'learning_rate': 4.800000000000001e-06, 'epoch': 0.44, 'step': 55}
12:34:28	{'loss': 0.7706, 'learning_rate': 4.3e-06, 'epoch': 0.48, 'step': 60}
12:34:28	{'loss': 0.8132, 'learning_rate': 3.8000000000000005e-06, 'epoch': 0.52, 'step': 65}
12:34:28	{'loss': 0.8094, 'learning_rate': 3.3000000000000006e-06, 'epoch': 0.56, 'step': 70}
12:34:28	{'loss': 0.7413, 'learning_rate': 2.8000000000000003e-06, 'epoch': 0.6, 'step': 75}
12:34:28	{'loss': 0.8326, 'learning_rate': 2.3000000000000004e-06, 'epoch': 0.64, 'step': 80}
12:34:28	{'loss': 0.7873, 'learning_rate': 1.8000000000000001e-06, 'epoch': 0.68, 'step': 85}
12:34:28	{'loss': 0.789, 'learning_rate': 1.3e-06, 'epoch': 0.72, 'step': 90}
12:34:28	{'loss': 0.8452, 'learning_rate': 8.000000000000001e-07, 'epoch': 0.76, 'step': 95}
12:34:28	{'loss': 0.8092, 'learning_rate': 3.0000000000000004e-07, 'epoch': 0.8, 'step': 100}
12:34:28	{'eval_loss': 0.744220495223999, 'eval_runtime': 0.0774, 'eval_samples_per_second': 64.586, 'eval_steps_per_second': 12.917, 'epoch': 0.8, 'step': 100}
12:34:28	{'train_runtime': 47.6549, 'train_samples_per_second': 33.575, 'train_steps_per_second': 2.098, 'total_flos': 114345191087424.0, 'train_loss': 0.8323853802680969, 'epoch': 0.8, 'step': 100}
12:34:28	Start Saving the model...
12:34:33	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 14 12:34:59 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
12:34:59	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
12:35:02	Using Locally Processed Dataset. Skip Processing...
12:35:02	Generating Seq2SeqTrainer Arguments
12:35:02	Generating Seq2SeqTrainer
12:35:02	Start Training...
{'loss': 0.9372, 'learning_rate': 9.600000000000001e-06, 'epoch': 0.04, 'step': 5}
{'loss': 0.9539, 'learning_rate': 9.200000000000002e-06, 'epoch': 0.08, 'step': 10}
{'loss': 0.9213, 'learning_rate': 8.700000000000001e-06, 'epoch': 0.12, 'step': 15}
{'loss': 0.874, 'learning_rate': 8.2e-06, 'epoch': 0.16, 'step': 20}
{'loss': 0.8497, 'learning_rate': 7.7e-06, 'epoch': 0.2, 'step': 25}
{'loss': 0.8623, 'learning_rate': 7.2000000000000005e-06, 'epoch': 0.24, 'step': 30}
{'loss': 0.7895, 'learning_rate': 6.700000000000001e-06, 'epoch': 0.28, 'step': 35}
{'loss': 0.8644, 'learning_rate': 6.200000000000001e-06, 'epoch': 0.32, 'step': 40}
{'loss': 0.8408, 'learning_rate': 5.7e-06, 'epoch': 0.36, 'step': 45}
{'loss': 0.7933, 'learning_rate': 5.2e-06, 'epoch': 0.4, 'step': 50}
{'loss': 0.7975, 'learning_rate': 4.7e-06, 'epoch': 0.44, 'step': 55}
{'loss': 0.7626, 'learning_rate': 4.2000000000000004e-06, 'epoch': 0.48, 'step': 60}
{'loss': 0.8135, 'learning_rate': 3.7e-06, 'epoch': 0.52, 'step': 65}
{'loss': 0.7836, 'learning_rate': 3.2000000000000003e-06, 'epoch': 0.56, 'step': 70}
{'loss': 0.762, 'learning_rate': 2.7000000000000004e-06, 'epoch': 0.6, 'step': 75}
{'loss': 0.8379, 'learning_rate': 2.2e-06, 'epoch': 0.64, 'step': 80}
{'loss': 0.738, 'learning_rate': 1.7000000000000002e-06, 'epoch': 0.68, 'step': 85}
{'loss': 0.7998, 'learning_rate': 1.2000000000000002e-06, 'epoch': 0.72, 'step': 90}
{'loss': 0.7802, 'learning_rate': 7.000000000000001e-07, 'epoch': 0.76, 'step': 95}
{'loss': 0.7681, 'learning_rate': 2.0000000000000002e-07, 'epoch': 0.8, 'step': 100}
{'eval_loss': 0.6385095715522766, 'eval_runtime': 0.0999, 'eval_samples_per_second': 50.062, 'eval_steps_per_second': 10.012, 'epoch': 0.8, 'step': 100}
{'train_runtime': 47.1112, 'train_samples_per_second': 33.962, 'train_steps_per_second': 2.123, 'total_flos': 115758656456256.0, 'train_loss': 0.8264809226989747, 'epoch': 0.8, 'step': 100}
12:36:15	Start Saving the model...
12:36:23	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 14 13:36:01 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
13:36:01	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
13:36:03	Using Locally Processed Dataset. Skip Processing...
13:37:29	Using model: ./model/trained_yifanhua-threadripper_Nov14_12:24
13:37:30	Input text: I'm loading the model from the Hugging Face Hub!

Start -- Nov 14 13:38:25 -- pytorch=2.1.0+cu121, device=cuda, cpu_count=48
13:38:25	Initializing pretrained model, processor, tokenizer, speaker_model, data_collator, and vocoder
13:38:28	Using Locally Processed Dataset. Skip Processing...
13:38:35	Using model: ./model/trained_yifanhua-threadripper_Nov13_19:26
13:38:36	Input text: I'm loading the model from the Hugging Face Hub!
18:02:05	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
18:13:36	Finish Loading Remote Dataset. Length of dataset: 12135
18:13:36	Using Mozilla Common Voice. Additional Cleaning Needed
18:13:36	Starting Size of Mozilla Common Voice: 12135
18:14:17	Size after filtering accent: 8671
18:14:44	Size after filtering gender: 8637
18:15:09	Size after filtering voting: 8637
18:15:10	Finish normalizing input text
18:15:10	Finish Cleaning Dataset. Length of dataset: 8637
18:15:10	Start Setting Sampling Rate to 16 kHz...
18:15:10	Setting Sampling Rate Successfully
19:19:17	Loading Remote Dataset: mozilla-foundation/common_voice_1_0. Sub-collection: en
19:30:51	Finish Loading Remote Dataset. Length of dataset: 12135
19:30:51	Using Mozilla Common Voice. Additional Cleaning Needed
19:30:51	Starting Size of Mozilla Common Voice: 12135
19:31:32	Size after filtering accent: 8671
19:31:58	Size after filtering gender: 8637
19:32:23	Size after filtering voting: 8637
19:32:24	Finish normalizing input text
19:32:24	Finish Cleaning Dataset. Length of dataset: 8637
19:32:24	Start Setting Sampling Rate to 16 kHz...
19:32:24	Setting Sampling Rate Successfully
19:32:25	Start Setting Sampling Rate to 16 kHz...
19:32:25	Setting Sampling Rate Successfully
20:28:01	Start Setting Sampling Rate to 16 kHz...
20:28:01	Setting Sampling Rate Successfully
20:45:18	Start Setting Sampling Rate to 16 kHz...
20:45:18	Setting Sampling Rate Successfully
20:46:56	Start Setting Sampling Rate to 16 kHz...
20:46:56	Setting Sampling Rate Successfully
20:47:55	Start Setting Sampling Rate to 16 kHz...
20:47:55	Setting Sampling Rate Successfully
20:48:28	Start Setting Sampling Rate to 16 kHz...
20:48:28	Setting Sampling Rate Successfully
20:55:09	Start Setting Sampling Rate to 16 kHz...
20:55:09	Setting Sampling Rate Successfully
20:55:58	Start Setting Sampling Rate to 16 kHz...
20:55:58	Setting Sampling Rate Successfully
20:56:39	Start Setting Sampling Rate to 16 kHz...
20:56:39	Setting Sampling Rate Successfully
20:57:10	Start Setting Sampling Rate to 16 kHz...
20:57:10	Setting Sampling Rate Successfully
20:58:09	Start Setting Sampling Rate to 16 kHz...
20:58:09	Setting Sampling Rate Successfully
20:59:50	Start Setting Sampling Rate to 16 kHz...
20:59:50	Setting Sampling Rate Successfully
21:00:30	Start Setting Sampling Rate to 16 kHz...
21:00:30	Setting Sampling Rate Successfully
21:02:35	Start Setting Sampling Rate to 16 kHz...
21:02:35	Setting Sampling Rate Successfully
21:05:19	Start Setting Sampling Rate to 16 kHz...
21:05:19	Setting Sampling Rate Successfully
21:09:08	Start Setting Sampling Rate to 16 kHz...
21:09:08	Setting Sampling Rate Successfully
21:10:09	Start Setting Sampling Rate to 16 kHz...
21:10:09	Setting Sampling Rate Successfully
21:10:25	Start Setting Sampling Rate to 16 kHz...
21:10:25	Setting Sampling Rate Successfully
21:12:37	Start Setting Sampling Rate to 16 kHz...
21:12:37	Setting Sampling Rate Successfully
21:13:49	Start Setting Sampling Rate to 16 kHz...
21:13:49	Setting Sampling Rate Successfully
21:21:56	Start Setting Sampling Rate to 16 kHz...
21:21:56	Setting Sampling Rate Successfully
21:22:05	Start Setting Sampling Rate to 16 kHz...
21:22:05	Setting Sampling Rate Successfully
21:24:42	Start Setting Sampling Rate to 16 kHz...
21:24:42	Setting Sampling Rate Successfully
21:25:03	Start Setting Sampling Rate to 16 kHz...
21:25:03	Setting Sampling Rate Successfully
21:25:52	Start Setting Sampling Rate to 16 kHz...
21:25:52	Setting Sampling Rate Successfully
21:27:08	Start Setting Sampling Rate to 16 kHz...
21:27:08	Setting Sampling Rate Successfully
21:27:48	Start Setting Sampling Rate to 16 kHz...
21:27:48	Setting Sampling Rate Successfully
21:30:01	Start Setting Sampling Rate to 16 kHz...
21:30:01	Setting Sampling Rate Successfully
